haystack/components/generators/chat/azure.py

```diff
@@ -32,30 +32,33 @@ class AzureOpenAIChatGenerator(OpenAIChatGenerator):
 
     messages = [ChatMessage.from_user("What's Natural Language Processing?")]
 
-    client = AzureOpenAIGenerator(azure_endpoint="<Your Azure endpoint e.g. `https://your-company.azure.openai.com/>",
-                                api_key=Secret.from_token("<your-api-key>"),
-                                azure_deployment="<this a model name, e.g. gpt-35-turbo>")
+    client = AzureOpenAIGenerator(
+        azure_endpoint="<Your Azure endpoint e.g. `https://your-company.azure.openai.com/>",
+        api_key=Secret.from_token("<your-api-key>"),
+        azure_deployment="<this a model name, e.g. gpt-35-turbo>")
     response = client.run(messages)
     print(response)
+    ```
 
-    >>{'replies': [ChatMessage(content='Natural Language Processing (NLP) is a branch of artificial intelligence
-    >>that focuses on enabling computers to understand, interpret, and generate human language in a way that is
-    >>meaningful and useful.', role=<ChatRole.ASSISTANT: 'assistant'>, name=None,
-    >>meta={'model': 'gpt-3.5-turbo-0613', 'index': 0, 'finish_reason': 'stop',
-    >>'usage': {'prompt_tokens': 15, 'completion_tokens': 36, 'total_tokens': 51}})]}
-
+    ```
+    {'replies':
+        [ChatMessage(content='Natural Language Processing (NLP) is a branch of artificial intelligence that focuses on
+         enabling computers to understand, interpret, and generate human language in a way that is meaningful and useful.',
+         role=<ChatRole.ASSISTANT: 'assistant'>, name=None,
+         meta={'model': 'gpt-3.5-turbo-0613', 'index': 0, 'finish_reason': 'stop',
+         'usage': {'prompt_tokens': 15, 'completion_tokens': 36, 'total_tokens': 51}})]
+    }
     ```
 
      Key Features and Compatibility:
-         - **Primary Compatibility**: Designed to work seamlessly with the OpenAI API Chat Completion endpoint
-            and gpt-4 and gpt-3.5-turbo family of models.
-         - **Streaming Support**: Supports streaming responses from the OpenAI API Chat Completion endpoint.
-         - **Customizability**: Supports all parameters supported by the OpenAI API Chat Completion endpoint.
+       - Primary Compatibility: Designed to work seamlessly with the OpenAI API Chat Completion endpoint and gpt-4 and gpt-3.5-turbo family of models.
+       - Streaming Support: Supports streaming responses from the OpenAI API Chat Completion endpoint.
+       - Customizability: Supports all parameters supported by the OpenAI API Chat Completion endpoint.
```

Comment:
this part seems to me almost a repetition of what has already been said in the initial description of the Component. Feel free to cut it or include it in the initial description.

Reply:
if it's there makes sense to be in the initial description, moved